addons/pos_restaurant/static/tests/tours/utils/merge_table_util.js

```diff
@@ -0,0 +1,55 @@
+import * as FloorScreen from "@pos_restaurant/../tests/tours/utils/floor_screen_util";
+import { TourError } from "@web_tour/tour_service/tour_utils";
+import * as ProductScreenPos from "@point_of_sale/../tests/tours/utils/product_screen_util";
+import * as ProductScreenResto from "@pos_restaurant/../tests/tours/utils/product_screen_util";
+
+const ProductScreen = { ...ProductScreenPos, ...ProductScreenResto };
+
+export function mergeTableHelpers(childName, parentName) {
+    return [
+        FloorScreen.clickTable(childName),
+        ProductScreen.clickControlButton("More"),
```

Comment:
```suggestion
        ProductScreen.clickControlButtonMore(),
```
Can you confirm if this suggestion is correct?

Reply:
yes, it is :)